projects/gateway2/translator/sslutils/ssl_utils.go

```diff
@@ -129,6 +129,8 @@ func ApplyMaximumTlsVersion(ctx context.Context, in string, out *ssl.SslConfig)
 func ApplyOneWayTls(ctx context.Context, in string, out *ssl.SslConfig) error {
 	if strings.ToLower(in) == "true" {
 		out.OneWayTls = wrapperspb.Bool(true)
+	} else if strings.ToLower(in) == "false" {
+		out.OneWayTls = wrapperspb.Bool(false)
 	}
 	return nil
```

Comment:
Though... if we are going to set it when both false and true, might as well bubble up an error here that it's an invalid value?